src/config.js

```diff
@@ -6,33 +6,31 @@ const config = cfg({
   allowedOrigins: {
     env: 'ALLOWED_ORIGINS',
     type: 'string',
-    default: '*',
+    required: true,
```

Comment:
CORS options probably shouldn't be required.

Reply:
If we make this "not required" then we'll need to either not use cors middleware when env vars are not provided or default somewhere to "*" for allowed headers and origins.

Reply:
Could you leave the default and set required to `false`? Or would that somehow be bad practice? @naptowncode?